language/lexer.py

```diff
@@ -0,0 +1,247 @@
+from langkit.lexer import (  # type: ignore
+    Alt,
+    Case,
+    Ignore,
+    Lexer,
+    LexerToken,
+    Literal,
+    Pattern,
+    WithSymbol,
+    WithText,
+    WithTrivia,
+)
+
+
+class Token(LexerToken):
+    # Identifiers
+    UnqualifiedIdentifier = WithSymbol()
+
+    # Keywords
+    Package = WithText()
+    Is = WithText()
+    If = WithText()
+    End = WithText()
+    Null = WithText()
+    Semicolon = WithText()
+    DoubleColon = WithText()
+    Assignment = WithText()
+    Colon = WithText()
```

Comment:
Just a sorting issue, but I think they would fit better in symbols than in keywords.

Reply:
Changed in 50032f5.